UI/window-basic-properties.cpp

```diff
@@ -166,6 +168,22 @@ OBSBasicProperties::OBSBasicProperties(QWidget *parent, OBSSource source_)
 
 	} else {
 		ui->preview->hide();
+		verticalMixer = false;
+	}
+
+	if ((caps & OBS_SOURCE_AUDIO) == OBS_SOURCE_AUDIO) {
```

Comment:
```suggestion
	if (obs_source_audio_active(source)) {
```

This'll check whether the source is actually set to play audio. Will require some experimentation to ensurte it works how we expect.

Reply:
Updated to handle sources such as browser and Windows video capture.